tests/headers.rs

```diff
@@ -104,13 +127,16 @@ fn check_imports() {
                 break;
             }
             let parts: Vec<&str> = line.split("&[").collect();
-            let deps = parts[1].split(',').map(|x| get_between_quotes(x).to_owned()).collect();
-            files_deps.insert(format!("{}.rs", get_between_quotes(parts[0])), deps);
+            files_deps.insert(format!("{}.rs", get_between_quotes(parts[0])),
+                              parts[1].split(',')
+                                      .map(|x| get_between_quotes(x).to_owned())
+                                      .filter(|x| !x.is_empty())
+                                      .collect());
```

Comment:
Why don't you like assigning deps to an intermediate variable first? It really helps with minimizing indentation.